src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -486,32 +486,15 @@ tr.status-warning td {
     padding-bottom: 99999px; {# needed for 'same-height columns' trick #}
     width: 220px;
 }
-#sidebar a {
-    color: #ADD8E6;
-}
```

Comment:
Nope. In the toolbar we use `.sf-toolbar-block .sf-toolbar-info-piece a`. I've tried to make the toolbar and the profiler as independent as possible.